lib/csv.rb

```diff
@@ -1782,9 +1826,19 @@ def lineno
     end
   end
 
-  #
-  # The last row read from this file.
-  #
+  # Returns the row most recently parsed:
```

Comment:
```suggestion
  # Returns the row most recently read:
```
This is probably just semantics, but in this case line has not been parsed yet, taking the example below, the difference per iteration in the loop make it clear:

|csv.line (just read) | row (parsed)|
|---|---|
|"foo,0\n"|["foo", "0"]|
|"bar,1\n"|["bar", "1"]|
| "baz,2\n"|["baz", "2"]|

Reply:
Changed to  "line most recently read" in next PR.